app/__init__.py

```diff
@@ -42,11 +42,13 @@ def create_app(config_name):
                 content_loader.load_manifest(framework_data['slug'], 'briefs', 'display_brief')
 
     from .main import main as main_blueprint
+    from .main import projects as projects_blueprint
```

Comment:
can we call it direct_award_projects, so this would be `from .main import direct_award_projects as projects_blueprint`